### PR TITLE
Fix Haxe3 & Add Tab

### PR DESCRIPTION
**Pull request by JbIPS**
I experienced some compilation problems with @freakinruben's compatibility code, so I fixed these and use the opportunity to add my code for Bootstrap's Tab.

Tested with this code, works like a charm:

``` haxe
var tab: JQuery = new JQuery(".nav-tabs a");
tab.tab();
tab.click(function(e){
    new JQuery(e.target).tabShow();
});
```

### CODE REVIEW

File: bootstrap/Lib.hx [hunk 1]
```diff
@@ -3,15 +3,16 @@ package bootstrap;
 import haxe.macro.Expr;
 import haxe.macro.Context;
 
-@:macro class Lib {
```
Comment (JbIPS) [inline bootstrap/Lib.hx:6]: Haxe 3 doesn't support macro class

**Comment by JbIPS** [comment]
If you have a minute, you could edit the readme to update the coverage of bootstrap elements (add tabs)

**Comment by clemos** [comment]
It's done.
But you could have done it too :p
Thanks again

**Comment by JbIPS** [comment]
Yeah, forgot it, and I wouldn't make another pull request just for that

**Comment by clemos** [comment]
I believe you can add it to the same pull request somehow, but I don't remember precisely how it works.
Nevermind... ;)

**Pull request merged**